CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
```

Comment:
I've started the following etherpads:

 - [ui process proposals](http://grimwire.com:9001/p/ui-process).
 - [feature proposals](http://grimwire.com:9001/p/pw-feature-ideas).

let's use these as a starting point